### PR TITLE
Support Nodejs 12 Agent

### PR DESCRIPTION
**Pull request by waveywaves**
- Add Support for Nodejs 12 Agent for Jenkins

### CODE REVIEW

File: agent-nodejs-12/Dockerfile.localdev [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0
+
+MAINTAINER Akram Ben Aissi <abenaiss@redhat.com>
```
Comment (akram) [inline agent-nodejs-12/Dockerfile.localdev:3]: I would like to change all these by:
MAINTAINER OpenShift Developer Services <openshift-dev-services+jenkins@redhat.com>

can we start with with this one?

File: agent-nodejs-12/test/agent_nodejs_test.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+var imageName string
+
+var _ = BeforeSuite(func() {
+	var err error
+	dockercli, err = docker.NewEnvClient()
+	Expect(err).NotTo(HaveOccurred())
+
+	imageName = os.Getenv("IMAGE_NAME")
+	if imageName == "" {
+		imageName = "vbobade/openshift-jenkins-base-agent"
```
Comment (akram) [inline agent-nodejs-12/test/agent_nodejs_test.go:29]: it is only visible in the e2e test right?

**Comment by nielsbril** [comment]
When will this image be available on Quay.io?

**Comment by akram** [comment]
/retitle Bug 1848284: Support Nodejs 12 Agent

**Pull request merged**